conformity/fields/basic.py

```diff
@@ -26,29 +26,43 @@ def introspect(self):
         raise NotImplementedError("You must override introspect() in a subclass")
 
 
-@attr.s
 class Constant(Base):
     """
-    Value that must match exactly.
+    Value that must match exactly. You can pass a series of options
+    and any will be accepted.
     """
 
-    value = attr.ib()
-    description = attr.ib(default=None)
+    def __init__(self, *args, **kwargs):
+        self.values = args
```

Comment:
Should we do `self.values = set(args)` here, so that the below `in` lookup is considerably faster (especially for fields with many constants, like a country code)?